kolibri/plugins/learn/assets/src/views/ChannelRenderer/CustomContentRenderer.vue

```diff
@@ -117,6 +117,12 @@
           getParams: {
             ids: options.ids,
             parent: options.parent === 'self' ? this.topic.id : options.parent,
+            channel_id: this.topic.id,
+            page: options.page ? options.page : 1,
+            page_size: options.pageSize ? options.pageSize : 50,
+            only_content: options.onlyContent ? true : undefined,
+            // Using Math.random as seed
+            random: options.random ? Math.random().toString() : undefined,
```

Comment:
I understand this is a cache-busting parameter, so can we use a timestamp here, instead?

Reply:
True, it's better to use a timestamp. I've changed it.